examples/helm-charts/cubejs/README.md

```diff
@@ -179,6 +179,7 @@ cubestore:
 | `global.devMode`                   | If true, enables development mode                                                                                               | `false` |
 | `global.debug`                     | If true, enables debug logging                                                                                                  | `false` |
 | `global.logLevel`                  | The logging level for Cube.js                                                                                                   | `warn`  |
+| `global.externalDefault`           | If true, uses Cube Store for storing Pre-aggregations                                                                           | `true`  |
```

Comment:
I'm not sure this is only for cubestore

Reply:
Nice catch! I'll edit this to mention other databases too.